resources/windows/wix/properties.wxs.erb

```diff
@@ -173,5 +173,26 @@
         Win64="<%= settings[:win64] %>" />
     </Property>
 
+    <!--
+      Check if there is a pre-existing Hiera 3 format file in either:
+        1. C:\ProgramData\PuppetLabs\code
+        2. C:\ProgramData\PuppetLabs\puppet\etc
+      If either file exists, then we need to block the installation of the new format
+      Hiera 5 default files.
+      This Programdata directory is fixed, so its not influenced by a change in
+      INSTALLDIR, so we can use the util:search property instead of a custom action.
+      These properties are used in appdatafiles.wxs.
+    -->
```

Comment:
For reference @branan & @MosesMendoza This is the detection/search for pre-existing files in windows which causes ```PUP_PREEXISTING_CODE_HIERA``` or ```PUP_PREEXISTING_CONF_HIERA``` to be populated.

These are used to drive a conditional for the components (see above) to block them installing if either of these properties exist.